setup.py

```diff
@@ -1,11 +1,41 @@
-#!/usr/bin/env python
+from setuptools import setup
 
-from distutils.core import setup
-from catkin_pkg.python_setup import generate_distutils_setup
+package_name = 'urdfdom_py'
 
-d = generate_distutils_setup(
+setup(
+    name=package_name,
+    version='0.3.3',
+    package_dir={'': 'src'},
     packages=['urdf_parser_py', 'urdf_parser_py.xml_reflection'],
-    package_dir={'': 'src'}
+    data_files=[
+        ('scripts/display_urdf',
+        ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    author='Víctor Mayoral Vilches',
+    author_email='vmayoral@acutronicrobotics.com',
+    maintainer='Víctor Mayoral Vilches',
```

Comment:
I don't mind adding you as a maintainer, but if you want to be the maintainer I'd suggest that we add you to the package.xml as well.  If you don't want to be a maintainer, you can just put my name/email here.